src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -86,87 +83,74 @@ class UniversalNavbar extends React.Component {
           }}
         >
           <Layout.ScrollDetector>
-            <Media.PhoneOnly>
-              {/* Hamburger button */}
-              <div style={{ zIndex: 2, position: 'fixed', top: 24, right: 24 }}>
-                <TransformingBurgerButton
-                  showMobileMenu={showMobileMenu}
-                  clickHandler={this.toggleHamburger}
-                />
-              </div>
```

Comment:
I don’t love BEM fwiw but for now I think it’s fine and I’m glad we’re on same page re converting to css modules and losing design-system.css. I’d also like to consider removing any need for bundle and use ES6 imports for CMS but I may be missing some things

Reply:
> I don’t love BEM fwiw but for now I think it’s fine and I’m glad we’re on same page re converting to css modules and losing design-system.css. 

I see BEM as basically manual CSS modules, so like CSS Modules > BEM > no convention. In general I'm very pro, but I worried we would run into trouble trying to define CSS modules in one place and consume them elsewhere. But I think we can figure it out.

> I’d also like to consider removing any need for bundle and use ES6 imports for CMS but I may be missing some things

Agreed.